config.js.docker

```diff
@@ -8,6 +8,29 @@ function getParam(name, _default) {
   return argv[name] || process.env[envname] || _default;
 }
 
+function getIntParam(name, _default) {
+  const param = getParam(name, _default);
+  const parsedParam = parseInt(param);
+  if (Number.isInteger(parsedParam)) {
```

Comment:
I feel `parseInt()` will always return an int. Should we prevent things like `"123a"`?

Reply:
Added this commit https://github.com/HathorNetwork/hathor-wallet-headless/pull/82/commits/8e1208b58469e937e96363ab4f25dc1c15b1c3bb. What do you think?